Gameplay.js

```diff
@@ -84,22 +84,24 @@ class Gameplay {
 		this.board1.render(document.getElementById("board1"), this, this.turn, preventClicking);
 	}
 
-	//FIX: doesn't show both boards
 	/**
 	* @description Render the boards, showing ships on both boards, and display a victory message
 	**/
 	gameEnd() {
 		alert("You win!") //Improve: Say which player won and display it better
 		this.board0.render(document.getElementById("board0"), this, true, true);
 		this.board1.render(document.getElementById("board1"), this, true, true);
+		
+		document.getElementById("switch-turn").disabled = true;
```

Comment:
It would probably make more sense to hide the button to be consistent with elsewhere